avaframe/runStandardTests.py

```diff
@@ -0,0 +1,114 @@
+"""
+    Run script for running Standalone DFA for the standard tests
+    This file is part of Avaframe.
+"""
+
+# Load modules
+import os
+
+# Local imports
+from avaframe.com1DFA import com1DFA
+from avaframe.log2Report import generateReport as gR
+from avaframe.log2Report import generateCompareReport
+from avaframe.out3SimpPlot import outPlotAllPeak as oP
+from avaframe.out3Plot import outQuickPlot
+from avaframe.in3Utils import fileHandlerUtils as fU
+from avaframe.in3Utils import cfgUtils
+from avaframe.in3Utils import logUtils
+import time
+
+# log file name; leave empty to use default runLog.log
+logName = 'runStandardTests'
+
+# Load avalanche directory from general configuration file
+cfgMain = cfgUtils.getGeneralConfig()
+#
+standardNames = ['data/avaBowl', 'data/avaFlatPlane', 'data/avaHelix', 'data/avaHelixChannel', 'data/avaHockey',
+                 'data/avaHockeySmoothChannel', 'data/avaHockeySmoothSmall', 'data/avaInclinedPlane']
+# standardNames = ['data/avaBowl', 'data/avaHelixChannel']
+
+# Set directory for full standard test report
+outDir = os.path.join(os.getcwd(), 'tests', 'reports')
+fU.makeADir(outDir)
+
+# Start writing markdown style report for standard tests
+with open(os.path.join(outDir, 'fullSimulationReport.md'), 'w') as pfile:
```

Comment:
rename it to standardTestReport? Following the runStandardTest naming...

Reply:
Done